### ISSUE TITLE
The term 'referand'

### ISSUE THREAD

**Issue by shiatsumat** [ericfindlay/lifetimes_presentation#1]
First of all, thank you for good slides. I would like to make a few comments.
I assume that the term 'referand' often used in the slides was coined based on 'operand', but it is not widely used.
I think that a more standard term is 'referent'.

**Comment by currency-engineering** [comment]
Thanks for the feedback. Others were also confused by this. I think it would be better if I used the word 'target'. Hopefully this should make it clearer.

**Comment by currency-engineering** [comment]
No, actually, I think 'referand' is ok, but its needs to be explicity defined in the presentation, possibly with an example of the stack. I think I'll add a slide to the presentation...

**Comment by shiatsumat** [comment]
The term 'target' sounds nice! I've never come up with that.
More instructive wording would be 'target data' / 'target address' I think.
